nss-day-cohort-56/rare-server-team-1-2
# Kl subscribe

Built Subscriptions serializer, views, urls

Added functionality to RareUsers to allow access to subscriptions

Type of change
Please delete options that are not relevant.

 New feature (non-breaking change which adds functionality)

How Can This Been Tested?
Run Server
Find a User that hasn't unsubscribed (may have to register a new user)
On PostMan run a POST server for the pk in subscriptions
Then GET subscriptions
Should return an empty array

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings
